scrapy/item.py

```diff
@@ -75,7 +81,10 @@ def __new__(cls, *args, **kwargs):
 
     def __init__(self, *args, **kwargs):
         self._values = {}
-        if args or kwargs:  # avoid creating dict for most common case
+        for k, v in self.fields.items():
```

Comment:
I don't think this is correct. It could end up assigning the _field itself_ as value:
```python
In [1]: from scrapy.item import Item, Field
   ...:
   ...: class CustomItem(Item):
   ...:     name = Field(serializer=str)
   ...:
   ...: i = CustomItem()

In [2]: type(i["name"])
Out[2]: scrapy.item.Field

In [3]: i["name"]
Out[3]: {'serializer': str}
```